frontend/src/sass/layout/_header.scss

```diff
@@ -1,6 +1,42 @@
 /* =================================
    LAYOUT / HEADER
    ================================= */
+.header {
```

Comment:
using classes makes it re-usable allowing not to rely on the HTML markup, if we needed to add a div around the header or change the layout a little we can just apply the  `.header` class to whatever is now considered a header. Ideally avoid styling the html elements directly unless it applies always to all elements of that type.